### PR TITLE
chore(lpa-trialists): set hillingdon as trialist and remove adur/telford

### PR DESCRIPTION
**Pull request by MrSimonEmms**
## Ticket Number

UCD-1090

## Description of change

Update trialists:
 - remove Adur
 - remove Telford and Wrekin
 - add Hillingdon

Tested on my cluster and working

This also updates the the `lpa-trialists.json` file (both test and in ConfigMap) to include the `horizonId`. The Hillingdon ID has been provided by Anthony

**IMPORTANT** this does not change the  in trial LPAs in development. They remain Adur and Telford & Wrekin

## Checklist

- [x] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [x] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

**Review by rcmsoftwaredevelopment — DISMISSED** [review]

**Review by Fenisay — APPROVED** [review]
I think we need also to update the screen "Only Wexford is available whatsoever" talking about the service but it can be done on the story anyway. If I'm the one taking it, I think I'll make this automatic to allow us to only to change this file next time

**Comment by MrSimonEmms** [comment]
> I think we need also to update the screen "Only Wexford is available whatsoever" talking about the service but it can be done on the story anyway. If I'm the one taking it, I think I'll make this automatic to allow us to only to change this file next time.
> 
> edit : "Important: this doesn't change the test LPAs in local development. They remain Adur and TWC"
> Ok then merry christmas, bye.

Yeah, that makes sense. I guess the wording is for the client to sign-off though - we're mere mortals in that regard.

**Comment by MrSimonEmms** [comment]
Have changed to get pushed to `master` so the change actually goes out on the clusters

**Review by Fenisay — APPROVED** [review]

**Review by a6software — DISMISSED** [review]

**Comment by MrSimonEmms** [comment]
Put back to `develop` as this will go out with Release 1 at end of sprint 6

**Review by a6software — DISMISSED** [review]

File: packages/appeals-service-api/data/lpa-trialists.json
Comment (MrSimonEmms) [inline packages/appeals-service-api/data/lpa-trialists.json:8]: This one doesn't correspond, but as outside the trial it won't be used

**Review by a6software — DISMISSED** [review]

**Review by Fenisay — APPROVED** [review]

**Comment by Fenisay** [comment]
Is this one not supposed to be a task merged in the parent ? Or at least that we should wait for the forms web app be merged into this one before merging onto the master ?

**Comment by MrSimonEmms** [comment]
@frank-dodion I'm closing this as it looks like you've done all the changes required for this in the `feature/UCD-1090-LPD-eligibility` branch. If there's anything I've missed, let me know

**Pull request closed without merging**
